README.md

```diff
@@ -33,6 +33,14 @@ Plex Rich Presence alternatives:
 *P.S.* Discord needs to be running on the machine Discrakt is running on.
 *P.P.S.* Place the `credentials.ini` file in the same directory as the executable.
 
+*P.P.P.S.* If you want to store the configuration in a common location, the `credentials.ini` can also be stored in:
+
+|Operating System|Location|Example|
+|--------|-----|-------|
+|Linux|`$XDG_CONFIG_HOME`/discrakt or `$HOME`/.config/discrakt|/home/alice/.config/discrakt/credentials.ini|
+|macOS|`$HOME`/Library/Application Support/discrakt|/Users/Alice/Library/Application Support/discrakt/credentials.ini|
+|Windows|`%LOCALAPPDATA%`\discrakt|C:\Users\Alice\AppData\Local\discrakt\credentials.ini|
```

Comment:
Isn't it this:
```suggestion
|Windows|`%LOCALAPPDATA%`\discrakt|C:\Users\Alice\AppData\Roaming\discrakt\credentials.ini|
```

Reply:
Why not go with Roaming instead of Local?
Not only is it easier to type `%AppData%` to navigate to it, but would also allow the config be backed up or migrated with the profile or even stay automatically synced across different devices in the same domain if the profile is attached to one.

Reply:
I'll be honest I didn't really know the difference between Local and Roaming! makes more sense to use Roaming then - cheers for mentioning